### PR TITLE
RCHAIN-2790 Make justifications visible with flag, sort validators in the graph

### PR DESCRIPTION
**Pull request by rabbitonweb**
## Overview
Make justifications visible with flag, sort validators in the graph

### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-2790

### CODE REVIEW

File: casper/src/main/scala/coop/rchain/casper/api/GraphGenerator.scala [hunk 1]
```diff
@@ -80,40 +83,27 @@ object GraphzGenerator {
...
-                        arrowHead = Some(NoneArrow)
-                      )
-                  )
-                parentsEdges *> justificationsEdges
-              }
-
-            }
+        _ <- config.showJustificationLines.fold(
+              drawJustificationDottedLines[G](g, validators),
+              ().pure[G]
```
Comment (lukasz-golebiewski) [inline casper/src/main/scala/coop/rchain/casper/api/GraphGenerator.scala]: Is this line necessary?

Comment (rabbitonweb) [reply casper/src/main/scala/coop/rchain/casper/api/GraphGenerator.scala]: yes
by default we will *NOT* draw justifictactions, only if `showJustificatioLines` flag is `true`
so here i'm folding over that flag, if `true` i draw, if `false` `(().pure[G])` which is a NOP

File: casper/src/main/scala/coop/rchain/casper/api/GraphGenerator.scala [hunk 2]
```diff
@@ -158,6 +148,47 @@ object GraphzGenerator {
       node = Map("width" -> "0", "height" -> "0", "margin" -> "0.03", "fontsize" -> "8")
     )
 
+  private def drawParentDependencies[G[_]: Applicative](
+      g: Graphz[G],
+      validators: List[(String, ValidatorsBlocks)]
```
Comment (lukasz-golebiewski) [inline casper/src/main/scala/coop/rchain/casper/api/GraphGenerator.scala]: Why does this method take `List[(String, ValidatorsBlocks)]` if enough would be `List[ValidatorsBlocks]`?

File: casper/src/main/scala/coop/rchain/casper/api/GraphGenerator.scala [hunk 3]
```diff
@@ -158,6 +148,47 @@ object GraphzGenerator {
...
+        case ValidatorBlock(blockHash, parentsHashes, _) => {
+          parentsHashes
+            .traverse(p => g.edge(blockHash, p, constraint = Some(false)))
+        }
+      }
+      .as(())
+
+  private def drawJustificationDottedLines[G[_]: Applicative](
+      g: Graphz[G],
+      validators: Map[String, ValidatorsBlocks]
```
Comment (lukasz-golebiewski) [inline casper/src/main/scala/coop/rchain/casper/api/GraphGenerator.scala]: Similar story here, you're using only the values so no need to pass the whole map with keys

**Pull request merged**